app/viewmodels/journeys/taskList/NationalInsuranceContributionsViewModel.scala

```diff
@@ -26,22 +28,27 @@ import viewmodels.journeys.{SummaryListCYA, getJourneyStatus}
 
 object NationalInsuranceContributionsViewModel {
 
-  def buildSummaryList(nationalInsuranceStatuses: List[JourneyNameAndStatus])(implicit messages: Messages): SummaryList = {
+  def buildSummaryList(nationalInsuranceStatuses: List[JourneyNameAndStatus], taxYear: TaxYear)(implicit messages: Messages): SummaryList = {
 
-    val nicRow = buildRow(NationalInsuranceContributions, nationalInsuranceStatuses, dependentJourneyIsFinishedForClickableLink = false)
+    val isAdjustmentsAnswered = JourneyStatus.getJourneyStatus(Adjustments, nationalInsuranceStatuses).isCompleted
+    val nicRow = buildRow(
+      NationalInsuranceContributions,
+      nationalInsuranceStatuses,
+      dependentJourneyIsFinishedForClickableLink = isAdjustmentsAnswered)(messages, taxYear)
```

Comment:
taxyear can be passed in as a no implicit argument, which means messages also doesn't need to be passed in the 2nd brackets (will do so implicitly)

Reply:
Done :)